search_queries_interval_rules_filter.go

```diff
@@ -0,0 +1,142 @@
+package elastic
+
+type FilterRule struct {
```

Comment:
It's too broad a term. We now have hundreds of types in `elastic` package already, so it'd be better to at least use a prefix to know where it belongs.